enos/enos-scenario-proxy.hcl

```diff
@@ -3,19 +3,86 @@
+  The scenario creates a new Vault Cluster using the candidate build and then runs the same Vault
+  build in Proxy mode and verifies behavior against the Vault cluster. The scenario also performs
+  standard baseline verification that is not specific to the Proxy mode deployment.
 
-    If you want to use the 'distro:leap' variant you must first accept SUSE's terms for the AWS
+  # How to run this scenario
+
+  1. Install Enos (more info: https://eng-handbook.hashicorp.services/internal-tools/enos/getting-started/)
+  
+  $ brew tap hashicorp/tap && brew update && brew install hashicorp/tap/enos
+
+  2. Authenticate to AWS with Doormat (more info: https://eng-handbook.hashicorp.services/internal-tools/enos/getting-started/#authenticate-to-aws-with-doormat)
+  
+  $ doormat login && eval $(doormat aws -a <your_aws_account_name> export)
+
+  3. Set the following variables either as environment variables (`export ENOS_VAR_var_name=value`)
+  or in your 'enos-local.vars' file:
+
+  Required variables
+    - aws_ssh_private_key_path (more info about AWS SSH keypairs: https://eng-handbook.hashicorp.services/internal-tools/enos/getting-started/#set-your-aws-key-pair-name-and-private-key)
+    - aws_ssh_keypair_name
+    - vault_build_date
+    - vault_product_version
+    - vault_revision
+
+  Optional variables
+    - aws_region (if different from the default value in enos-variables.hcl)
+    - consul_license_path (if using an ENT edition of Consul)
+    - distro_version_<distro> (if different from the default version for your target
+    distro; see list of supported distros and versions in enos-globals.hcl)
+    - vault_license_path (if using an ENT edition of Vault)
+  
+  4. Choose one of the following ways to get a Vault artifact and set the appropriate variables.
+  This will be the Vault artifact that we upgrade to; 'matrix.initial_version' will be the Vault
+  version we start from.
+
+    a. 'artifact_source:crt'
+    This will use a Vault artifact that you have already downloaded. Set the following variable:
+
+    ENOS_VAR_vault_artifact_path=path/to/existing-vault-artifact.[zip | rpm | deb]
+
+    b. 'artifact_source:local'
+    This will build a Vault .zip bundle from your local branch. Set the following variable:
+
+    ENOS_VAR_vault_artifact_path=path/to/where/vault-should-be-built.zip
+
+    c. 'artifact_source:artifactory'
+    This will download a Vault artifact of the specified version, edition, and revision SHA
+    from the 'stable' repo in Artifactory. Set the following variables:
+
+    export ENOS_VAR_artifactory_username=your-user
+    export ENOS_VAR_artifactory_token=your-token
+  
+  5. Choose the matrix variants you want to use, and launch the scenario with the appropriate
+  filter for those variants, e.g.:
+
+  $ enos scenario launch proxy arch:arm64 artifact_source:artifactory artifact_type:bundle backend:raft config_mode:file consul_edition:ce consul_version:1.17.0 distro:leap edition:ent seal:shamir
```

Comment:
Same recommendation as above: code block and newlines will make this more readable

Reply:
I can see your point but I think we will leave this as one-liner for the purpose of the docs -- formatting changes may vary by personal preference and also be shell dependent.